stingray/pulse/tests/test_search.py

```diff
@@ -111,6 +111,13 @@ def test_plot_phaseogram_direct(self):
         plt.savefig("phaseogram_direct.png")
         plt.close(plt.gcf())
 
+    def test_search_wrong_key_fails(self):
+        with pytest.raises(
+            ValueError,
+            match="Unidentified keyword(s) to fold_events: fdot, fddot \n Please refer to the description of the function for optional parameters.",
```

Comment:
```suggestion
            match="Unidentified keyword(s) to fold_events: fdot, fddot",
```

Its better to match the smallest possible part of the regex that makes it really specific. Try to run it locally in your computer, with the `--run-slow` option, to verify that it works

Reply:
The brackets were causing that issue. I fixed it and the test is working on my pc.